tools/accuracy_checker/accuracy_checker/metrics/semantic_segmentation.py

```diff
@@ -217,11 +216,16 @@ def update(self, annotation, prediction):
         result = np.zeros(shape=self.classes)
 
         annotation_data = annotation.mask
-        prediction_data = np.argmax(prediction.mask, axis=0) if self.use_argmax else prediction.mask.astype('int64')
+        prediction_data = prediction.mask
+
+        if prediction_data.shape[0] != 1:
```

Comment:
the case where (h, w) wihout separated dim for channel is also should be acceptable